sunpy/net/dataretriever/tests/test_norh.py

```diff
@@ -1,10 +1,14 @@
 import pytest
 
+from unittest import mock
 import astropy.units as u
 from astropy.time import TimeDelta
 from astropy.time import Time
 
+from datetime import datetime, timedelta
+from sunpy.time import parse_time
 from sunpy.time.timerange import TimeRange
+from sunpy.net.download import Results
 from sunpy.net.dataretriever.client import QueryResponse
```

Comment:
The imports you have added here will need to be reorganised.

We try to keep it 

```
standard python library

other packages

astropy

sunpy
```

Reply:
Sure, I will make a note for future PRs.